google/cloud/pubsub/internal/default_pull_ack_handler.cc

```diff
@@ -36,7 +36,22 @@ DefaultPullAckHandler::DefaultPullAckHandler(CompletionQueue cq,
       ack_id_(std::move(ack_id)),
       delivery_attempt_(delivery_attempt),
       lease_manager_(
-          MakePullLeaseManager(cq_, stub_, subscription_, ack_id_, options)) {}
+          MakePullLeaseManager(cq_, stub_, subscription_, ack_id_, options)) {
+  initialize();
+}
+
+DefaultPullAckHandler::DefaultPullAckHandler(
+    CompletionQueue cq, std::weak_ptr<SubscriberStub> w, Options const&,
```

Comment:
Are you planning to use this `Options const&` parameter  in a future PR?  If not, consider removing it.

Reply:
Ack, removed.